share/autostart

```diff
@@ -121,7 +121,10 @@ hc set frame_transparent_width 5
 hc set frame_gap 4
 
 hc attr theme.title_height 15
-hc attr theme.title_font '-*-fixed-medium-r-*-*-13-*-*-*-*-*-*-*'
```

Comment:
Do we want to keep of all of those added lines?

Reply:
For me, it is always quite hard to get the format of font specifications right because there are strange subtleties in every application. That's why I thought it might be helpful to see a few examples. But maybe it's better to move it to the documentation (I don't know where yet)

Reply:
I know. Maybe reduce it down to two? And is the default case a font that is always installed?

Reply:
Looks good to me now